dbus_service.py

```diff
@@ -458,49 +548,40 @@ def _update(self):
             self._refresh_data()
 
-            pre = "/Ac/" + self.pvinverterphase
-
             if self.is_data_up2date():
-                (power, pvyield, current, voltage) = self.get_values_for_inverter()
-
                 if self.dry_run:
                     logging.info("DRY RUN. No data is sent!!")
                 else:
-                    self._dbusservice[pre + "/Voltage"] = voltage
-                    self._dbusservice[pre + "/Current"] = current
-                    self._dbusservice[pre + "/Power"] = power
-                    self._dbusservice["/Ac/Power"] = power
-                    if power > 0:
-                        self._dbusservice[pre + "/Energy/Forward"] = pvyield
-                        self._dbusservice["/Ac/Energy/Forward"] = pvyield
-
-                logging.debug("Inverter #%d Power (/Ac/Power): %s", self.pvinverternumber, power)
-                logging.debug("Inverter #%d Energy (/Ac/Energy/Forward): %s", self.pvinverternumber, pvyield)
-                logging.debug("---")
-
+                    self.set_dbus_values()
             self._update_index()
             successful = True
         except requests.exceptions.RequestException as exception:
             if self.last_update_successful:
-                logging.warning(f"HTTP Error at _update for inverter {self.pvinverternumber} ({self._get_name()}): {str(exception)}")
+                logging.warning(f"HTTP Error at _update for inverter \
+                    {self.pvinverternumber} ({self._get_name()}): {str(exception)}")
         except ValueError as error:
             if self.last_update_successful:
-                logging.warning(f"Error at _update for inverter {self.pvinverternumber} ({self._get_name()}): {str(error)}")
-        except Exception as error:
+                logging.warning(f"Error at _update for inverter \
+                    {self.pvinverternumber} ({self._get_name()}): {str(error)}")
+        except Exception as error: # pylint: disable=broad-except
             if self.last_update_successful:
-                logging.warning(f"Error at _update for inverter {self.pvinverternumber} ({self._get_name()})", exc_info=error)
+                logging.warning(f"Error at _update for inverter \
+                    {self.pvinverternumber} ({self._get_name()})", exc_info=error)
         finally:
             if successful:
                 if not self.last_update_successful:
-                    logging.warning(f"Recovered inverter {self.pvinverternumber} ({self._get_name()}): "
-                                    f"Successfully fetched data now: {'NOT (yet?)' if not self.is_data_up2date() else 'Is'} up-to-date")
+                    logging.warning(
+                        f"Recovered inverter {self.pvinverternumber} ({self._get_name()}): " +
+                        f"Successfully fetched data now: \
+                        {'NOT (yet?)' if not self.is_data_up2date() else 'Is'} up-to-date"
```

Comment:
This linebreak is wrong: It prints out too many spaces.

Reply:
changed that :-)